examples/framesjs-starter/app/examples/simplified-api/[[...routes]]/route.tsx

```diff
@@ -0,0 +1,72 @@
+import { AnyJson } from "../new-api";
+import {
+  FrameImage,
+  Frame,
+  FramesApp,
+  FrameLink,
+  renderFramesToResponse,
+} from "../react-api";
+
+type FrameState = {
+  clicked: number;
+};
+
+function isFrameState(
+  frameState: AnyJson | undefined
+): frameState is FrameState {
+  if (frameState && typeof frameState === "object" && "clcked" in frameState) {
+    return typeof frameState.clicked === "number";
+  }
+
+  return false;
+}
+
+export const framesApp = (
+  <FramesApp>
+    <Frame index>
+      {({ frameState }) => (
```

Comment:
@davidfurlong  @stephancill  if you need to access the frame state (state passed from button click!, not the global frames state parsed from meta tag) you can use a function to render the frame.

Reply:
@davidfurlong who is our target audience? I'm confused because render props is common pattern as well. But I would dive that much into React as it is unnecessary to build frames app and makes the app and whole build bigger.